gradle/libs.versions.toml

```diff
@@ -101,6 +101,6 @@
     kotlin-android = { id = "org.jetbrains.kotlin.android", version.ref = "kotlin" }
     sonarqube = { id = "org.sonarqube", version.ref = "sonar-gradle-plugin" }
     jacoco = { id = "org.kordamp.gradle.jacoco", version.ref = "jacoco" }
-    navigation-safe-args = { id = "androidx.navigation.safeargs.kotlin", version.ref = "navigation" }
+    navigation-safe-args = { id = "androidx.navigation.safeargs.kotlin", version = "2.5.3" }
```

Comment:
Any reason `2.5.3` is hard-coded here and not declared above in `[versions]`?

Reply:
> Any reason `2.5.3` is hard-coded here and not declared above in `[versions]`?

Yeah, apparently the plugin version starts to diverge from the library version so they aren't shipped together. But indeed, the value can be defined in the versions for consistency. I will update that.